t/js_saml.t

```diff
@@ -41,7 +41,7 @@ select STDOUT; $| = 1;
 eval { require JSON::PP; };
 plan(skip_all => "JSON::PP not installed") if $@;
 
-my $t = Test::Nginx->new()->has(qw/http rewrite proxy gzip api keyval/)
+my $t = Test::Nginx->new()->has(qw/http rewrite proxy gzip/)
```

Comment:
I do not see any reason to change this. You still use these modules.

Reply:
You are right, for the `api` and `keyval` modules, specific regular expressions are used that check for the absence of the `--without-http_api_module` and `--without-http_keyval_module` flags, respectively.
However, the fact is that this is only present in the `se` branch; we use the open source test lib here.

Reply:
Got it.